lib/compat/wordpress-6.2/class-gutenberg-rest-global-styles-controller-6-2.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Global_Styles_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Base Global Styles REST API Controller.
+ */
+class Gutenberg_REST_Global_Styles_Controller_6_2 extends Gutenberg_REST_Global_Styles_Controller {
```

Comment:
```suggestion
class Gutenberg_REST_Global_Styles_Controller_6_2 extends WP_REST_Global_Styles_Controller {
```

We should apply this change after rebasing on top of the current trunk. The `Gutenberg_REST_Global_Styles_Controller` class was removed in #46102.

Reply:
Done - I also had to port across the `prepare_item_for_response` method so that the change to `WP_Theme_JSON` for handling `styles.css` was picked up